src/auto/functions.rs

```diff
@@ -57,7 +58,7 @@ pub fn assertion_message_expr<'a, 'b, P: Into<Option<&'a str>>, Q: Into<Option<&
 }
 
 //#[deprecated]
-//pub fn atexit(func: /*Unknown conversion*//*Unimplemented*/VoidFunc) {
+//pub fn atexit<P: FnOnce() + Send + Sync + 'static>(func: P) {
```

Comment:
`FnOnce() + Sync` makes no sense btw :) Why is this not generated though? (And it should not be generated, it would be unsafe)

Reply:
Because there is no user data. :)

Reply:
Ah, makes sense :) We *could* generate something with `fn()` instead of `Fn*()`, i.e. a raw function... in such cases. But not worth worrying about that here :)